src/components/taskDetails/ProgressUpdateCard/LatestProgressUpdateCardPresentation.tsx

```diff
@@ -65,21 +62,23 @@ export default function LatestProgressUpdateCardPresentation({
                     className={
                         styles['latest-progress-update-card__date-container']
                     }
-                    onMouseOver={onHoverOnDate}
-                    onMouseOut={onMouseOutOnDate}
                 >
-                    <FaRegClock />
-                    <span
-                        className={styles['progress-update-card__date-text']}
-                        data-testid="latest-progress-update-card-date"
-                    >
-                        {dateInAgoFormat}
-                    </span>
                     <Tooltip
-                        isVisible={isTooltipVisible}
-                        textToShow={tooltipText}
+                        content={tooltipText}
                         tooltipPosition={{ top: '-2.6rem', right: '-4rem' }}
-                    />
+                    >
+                        <span className={styles['date-clock-container']}>
+                            <FaRegClock />
+                            <span
+                                className={
+                                    styles['progress-update-card__date-text']
+                                }
+                                data-testid="latest-progress-update-card-date"
+                            >
+                                {dateInAgoFormat}
+                            </span>
+                        </span>
```

Comment:
Why is this part is under tooltip?

Reply:
See https://ant.design/components/tooltip for refrence